src/ui/screenshot_menu.ts

```diff
@@ -326,16 +401,28 @@ export class ScreenshotDialog extends Overlay {
     this.statisticsTable.classList.add(
       "neuroglancer-screenshot-statistics-table",
     );
-    this.statisticsTable.title = "Screenshot statistics";
+    this.statisticsTable.title = "Screenshot progress";
 
     const headerRow = this.statisticsTable.createTHead().insertRow();
     const keyHeader = document.createElement("th");
-    keyHeader.textContent = "Screenshot statistics";
+    keyHeader.textContent = "Screenshot progress";
     headerRow.appendChild(keyHeader);
     const valueHeader = document.createElement("th");
     valueHeader.textContent = "";
     headerRow.appendChild(valueHeader);
 
+    const descriptionRow = this.statisticsTable.createTHead().insertRow();
+    const descriptionkeyHeader = document.createElement("th");
+    descriptionkeyHeader.colSpan = 2;
+    
+    descriptionkeyHeader.textContent = "Screenshot will take when all the chunks are loaded. If GPU memory is full, screenshot will only take the successfully loaded chunks.";
+    
+    const descriptionLearnMoreLink = document.createElement("a");
+    descriptionLearnMoreLink.text = "Learn more";
+    
+    descriptionkeyHeader.appendChild(descriptionLearnMoreLink);
```

Comment:
This docs page is not ready yet, so let's comment out creating the learn more link, and put a comment above it that this can be used to point to a docs page when complete